tests/foreman/cli/test_contentview.py

```diff
@@ -3962,3 +3962,23 @@ def test_positive_inc_update_should_not_fail(self, module_org):
         assert result[2]
         content_view = ContentView.info({'id': content_view['id']})
         assert '1.1' in [cvv_['version'] for cvv_ in content_view['versions']]
+
+    @pytest.mark.tier1
+    def test_positive_pulp_workers_running_state(self, target_sat):
```

Comment:
Consider making this part of the pulp service assertions done here:
https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/sys/test_pulp3_filesystem.py#L65

This isn't really related to contentview CLI.